src/docs/truffle/getting-started/installation.md

```diff
@@ -4,17 +4,95 @@ layout: docs.hbs
 ---
 # Installation
 
+## Requirements
+
+* [Node.js v12 - v16](#install-nodejs)
+* Windows, Linux, or macOS
+
+## Install Node.js
+
+### Linux and macOS
+
+Node Package Manager (NPM) recommends [installing Node.js with Node Version Manager (`nvm`)](https://node.dev/post/installing-node-js-tutorial-using-nvm-on-mac-os-x-and-ubuntu)
+to avoid permission errors when installing globally.
+
+1. Use `curl` or `wget` to install `nvm`:
+
+    === "curl"
+
+        ```bash
+        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+        ```
+
+    === "wget"
+
+        ```bash
+        wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+        ```
+
+1. Confirm that `nvm` installed correctly. You may need to reload the terminal for
+    the command to work:
+
+    ```bash
+    nvm --version
+    ```
+ 
+
+1. Truffle requires `node-gyp` for compiling native addon modules for Node.js.  Node-gyp in
+    turn has [dependencies](https://github.com/nodejs/node-gyp) which may be missing, causing the Truffle install to fail.
+    Truffle recommends installing the following to avoid getting dependency errors:
```

Comment:
```suggestion
1. Truffle requires `node-gyp` for compiling native add-on modules for Node.js. 
    Truffle recommends installing the following `node-gyp` [dependencies](https://github.com/nodejs/node-gyp) to avoid errors when installing Truffle:
```